src/lib/es5.d.ts

```diff
@@ -780,7 +780,7 @@ interface Date {
     /** Gets the day-of-the-month, using Universal Coordinated Time (UTC). */
     getUTCDate(): number;
     /** Gets the day of the week, using local time. */
-    getDay(): number;
+    getDay(): 0 | 1 | 2 | 3 | 4 | 5 | 6;

```

Comment:
This is incorrect. It can also be `NaN` (like every other method).

Reply:
Related: #15135